src/test/java/seedu/address/model/ModelManagerTest.java

```diff
@@ -40,14 +40,14 @@ public void setUserPrefs_nullUserPrefs_throwsNullPointerException() {
     @Test
     public void setUserPrefs_validUserPrefs_copiesUserPrefs() {
         UserPrefs userPrefs = new UserPrefs();
-        userPrefs.setAddressBookFilePath(Paths.get("address/book/file/path"));
+        userPrefs.setLibTaskFilePath(Paths.get("address/book/file/path"));
```

Comment:
Not sure if the path should still be ```address/book/file/path``` but the changes in this file looks good!

Reply:
This is just a random name for the test so I just leave it there.